packages/console/src/components/SubmitFormChangesActionBar/index.tsx

```diff
@@ -14,7 +14,13 @@ const SubmitFormChangesActionBar = ({ isOpen, isSubmitting, onDiscard }: Props)
 
   return (
     <div className={styles.container}>
-      <Button size="medium" title="general.discard" onClick={onDiscard} />
+      <Button
+        size="medium"
+        title="general.discard"
+        onClick={() => {
+          onDiscard();
+        }}
```

Comment:
ditto: onClick = {onDiscard}

Reply:
No, the `onClick` will pass an event to the called functions, which is not what we want.

<img width="584" alt="image" src="https://user-images.githubusercontent.com/10806653/202664541-5ec2e326-2d9b-4079-b0fa-7cd1b4660581.png">